deployments/k8s/khaas/README.md

```diff
@@ -0,0 +1,9 @@
+# KubeHound as a Service - ingestor - k8s deployment
+
+To deploy KHaaS ingestor services in a Kubernetes environment please refer to [k8s-deployment](https://kubehound.io/user-guide/khaas-101/#k8s-deployment)
+
+All the helms charts and templates as provided as example. You should tweak it to your own environment (resources limitation, endpoint configuration, ...). This will depend of the number of clusters and their size you want to ingest.
```

Comment:
```suggestion
All the Helm charts and templates as provided as example. You should tweak it to your own environment (resources limitation, endpoint configuration, ...). This will depend of the number of clusters and their size you want to ingest.
```